tests/mail/tests.py

```diff
@@ -393,6 +426,28 @@ def test_to_in_headers_only(self):
         ).message()
         self.assertEqual(message["To"], "to@example.com")
 
+    def test_to_not_repeated(self):
+        """
+        Make sure that to is not repeated (#35033)
+        """
+        email = EmailMessage(
+            subject="test subject",
+            body="test body",
+            from_email="from@example.com",
+            to=["guy1@example.com"],
+            headers={
+                "To": ", ".join(
+                    ["guy1@example.com", "guy2@example.com", "guy3@example.com"]
+                ),
```

Comment:
There is no need to use `join()` for static strings
```suggestion
                "To": "person1@example.com, person2@example.com",
```